llvm/test/SYCLLowerIR/ConditionalCallOnDevice/conditional-call-on-device-2.ll

```diff
@@ -42,8 +42,8 @@ entry:
   ret void
 }
 
-; CHECK: declare spir_func void @call_if_on_device_conditionally1_PREFIX_1(ptr, ptr, i32, i32)
-; CHECK: declare spir_func void @call_if_on_device_conditionally2_PREFIX_2(ptr, ptr, i32, i32)
+; CHECK: declare spir_func void @call_if_on_device_conditionally1_PREFIX_1(ptr[[VAL1:.*]], ptr[[VAL2:.*]], i32[[VAL3:.*]], i32[[VAL4:.*]])
```

Comment:
May be a test can be a bit more comprehensive with the actual checks for the attributes.